iac-common/src/main/java/org/sonar/iac/common/parser/grammar/Punctuator.java

```diff
@@ -36,6 +36,7 @@ public enum Punctuator implements GrammarRuleKey {
   RPARENTHESIS(")"),
   OR("||"),
   AND("&&"),
+  COALESCE("??"),
```

Comment:
TIL that this has a name...

Reply:
To be honest I found this yesterday in Bicep azure documentation :sweat_smile: